scalyr_agent/__scalyr__.py

```diff
@@ -117,7 +117,7 @@ def __determine_package_root():
             file_path = os.path.join(base, file_path)
         file_path = os.path.dirname(os.path.realpath(file_path))
     else:
-        return os.path.dirname(text_type(sys.executable, sys.getfilesystemencoding()))
```

Comment:
Is there a specific reason why the second argument (encoding) has been removed?

Reply:
on python3 `sys.executable` returns `str`(unicode), end `text_type` is `str` too, so `str(str(), encodeing="some_encoding")` will lead to error because we can not decode `str` from `str`.
But I agree that it might be wrong to remove it. I added a condition to handle this.